.github/workflows/publish.yaml

```diff
@@ -0,0 +1,18 @@
+name: '🚀 Publish Plugin'
+
+on:
+  release:
+    types:
+      - released
+  workflow_dispatch:
+
+jobs:
+  call:
+    uses: jellyfin/jellyfin-meta-plugins/.github/workflows/publish.yaml@master
+    with:
+      version: ${{ github.event.release.tag_name }}
+      is-unstable: ${{ github.event.release.prerelease }}
+    secrets:
+      deploy-host: ${{ secrets.DEPLOY_HOST }}
+      deploy-user: ${{ secrets.DEPLOY_USER }}
+      deploy-key: ${{ secrets.DEPLOY_KEY }}
```

Comment:
These will get ignored by any Org external forks of this template, as the steps using these won't be triggered by Org external runs of this